source/openapi-admin-v3.yaml

```diff
@@ -1989,6 +1989,41 @@ paths:
       responses:
         "404":
           description: No IP addresses or CIDR blocks to delete
+  "/groups/{groupId}/apps/{appId}/metrics":
+    get:
+      tags:
+        - metrics
```

Comment:
Let's also add an entry for the `metrics` tag to `components.tags` near the bottom of the file.